spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -348,7 +336,7 @@ def setup_common_actions(self):
         )
         # Filters
         self.filters_action = create_action(
-            self, _("Show files with these extensions..."), None,
+            self, _("Edit filter settings"), None,

```

Comment:
```suggestion
            self, _("Edit filter settings..."), None,
```

Per standard UI convention and as used elsewhere in Spyder, the `Edit filter settings` item should have an ellipses (`...`) at the end as it did previously, indicating that it opens a dialog that requires further interaction.